src/main/scala/temple/builder/DatabaseBuilder.scala

```diff
@@ -59,10 +59,15 @@ object DatabaseBuilder {
     readable: Metadata.Readable = Metadata.Readable.This,
     selectionAttribute: String = "id",
   ): ListMap[CRUD, Statement] = {
-    val tableName = StringUtils.snakeCase(serviceName)
-    val columns   = attributes.keys.map(Column).toSeq
+    val tableName = snakeCase(serviceName)
+    val columns   = attributes.keys.map(snakeCase(_)).map(Column).toSeq
     val providedColumns =
-      attributes.filter { case (_, attr) => attr != IDAttribute && attr != CreatedByAttribute }.keys.map(Column).toSeq
+      attributes
+        .filter { case (_, attr) => attr != IDAttribute && attr != CreatedByAttribute }
+        .keys
+        .map(snakeCase(_))
```

Comment:
Is there a reason why you’ve included the underscore here, rather than just the bare method name? Or conversely, why the maps were kept separate rather than `Column(snakeCase(_))`?

Reply:
`snakeCase` takes two arguments (albeit one with a default argument), so you can't just use the bare method name.

`Column(snakeCase(_))` sadly doesn't work without naming the argument, so `(name => Column(snakeCase(name))` if you think that's nicer?